completion/available/gradle.completion.bash

```diff
@@ -64,8 +64,9 @@ __gradle-set-files-checksum() {
 
 __gradle-generate-script-cache() {
 	# Invalidate cache after 3 weeks by default
-	local cache_ttl_mins=${GRADLE_CACHE_TTL_MINUTES:-30240}
-	local script_exclude_pattern=${GRADLE_COMPLETION_EXCLUDE_PATTERN:-"/(build|integTest|out)/"}
+	local cache_ttl_mins script_exclude_pattern gradle_build_scripts
```

Comment:
The local variable `gradle_build_scripts` are doubly declared. Either one should be removed.

Reply:
Sorry, missed these when resolving merge conflicts. Probably should have started from scratch